src/django/api/models/submission.py

```diff
@@ -41,6 +41,10 @@ def clean(self):
     def __str__(self):
         return f"Submission {self.pk} for {self.boundary}"
 
+    @property
+    def primary_contact(self):
+        return self.created_by or self.submitted_by
```

Comment:
This may always return `created_by` since that field is NOT NULL, and will always short circuit.

Let's just use `created_by` directly.